_posts/2023-07-05-niklas-fuzzing.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Niklas Gögge on Fuzz Testing"
+permalink: /blog/2023/07/05/fuzzing/
+layout: post
+author: brink
+name: "brink"
+alt: Niklas Gögge on Fuzz Testing
+category: "News"
```

Comment:
Is this the right category?

Reply:
I dont think that category is currently used, but yes, this would be in the technical category, Ill update